pages/previewPage.html

```diff
@@ -173,6 +226,28 @@
             } else {
               setZoomLevel(zoomLevel);
             }
+            window.postMessage({ command: "hideBusy" });
+            break;
+          case "showBusy":
+            timer = window.setTimeout(() => {
+              busyIndicator.style.visibility = "visible";
+            }, 2000);
```

Comment:
```suggestion
            }, 0);
```
with it smaller and at the bottom I think it is better for this to be more responsive

Reply:
I just removed the timer, since a timeout of zero is immediate.